ReportingServicesTools/Functions/Admin/Set-RsUrlReservation.ps1

```diff
@@ -13,9 +13,6 @@ function Set-RsUrlReservation
         .PARAMETER ReportServerVirtualDirectory
             Specify the name of the virtual directory for the Report Server Endpoint the default is ReportServer, it will configure it as http://myMachine/reportserver
         
-        .PARAMETER SenderAddress
```

Comment:
You should bring this back and rename the SenderAddress to PortalVirtualDirectory